contracts/UsingWitnet.sol

```diff
@@ -3,26 +3,23 @@
 pragma solidity >=0.7.0 <0.9.0;
 pragma experimental ABIEncoderV2;
 
-import "./Request.sol";
-import "./Witnet.sol";
-import "./WitnetRequestBoardInterface.sol";
-
+import "./libs/Witnet.sol";
+import "./interfaces/WitnetRequestBoardInterface.sol";
 
 /**
  * @title The UsingWitnet contract
  * @notice Contract writers can inherit this contract in order to create Witnet data requests.
  */
 abstract contract UsingWitnet {
-  using Witnet for Witnet.Result;
-
-  WitnetRequestBoardInterface internal immutable wrb;
+  WitnetRequestBoardInterface internal immutable witnet;
 
  /**
   * @notice Include an address to specify the WitnetRequestBoard.
   * @param _wrb WitnetRequestBoard address.
-  */
+  **/
   constructor(address _wrb) {
-    wrb = WitnetRequestBoardInterface(_wrb);
+    require(_wrb != address(0), "UsingWitnet: null contract");
```

Comment:
"WitnetRequestBoard: null contract"? and add one test for this require?

Reply:
Great idea. I will add a test case within the `test/using_witnet.js` testing suite.